lib/screens/import from seed/import_from_seed.dart

```diff
@@ -0,0 +1,288 @@
+                    });
+                  },
+                  icon: Icon(
+                      color: Colors.white,
+                      _confirmpassword
+                          ? Icons.remove_red_eye_outlined
+                          : Icons.visibility_off_outlined),
+                ),
+              ),
+              Container(
+                margin:
+                    const EdgeInsets.symmetric(horizontal: 32, vertical: 12),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    Text(
+                      'Sign in with Face ID?',
+                      style: Theme.of(context).textTheme.displaySmall,
+                    ),
+                    Transform.scale(
+                      scale: 1.2,
+                      child: Switch(
+                        inactiveThumbColor: const Color(0xFF181E25),
+                        inactiveTrackColor: const Color(0xFF384657),
+                        activeColor: Colors.white,
+                        trackOutlineColor:
+                            const MaterialStatePropertyAll(Colors.transparent),
+                        activeTrackColor: const Color(0xFF3D8DFF),
+                        value: _togglevalue,
+                        onChanged: (onChanged) {
+                          setState(
+                            () {
+                              _togglevalue = onChanged;
+                            },
+                          );
+                        },
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+              Container(
+                  margin: const EdgeInsets.only(left: 32, right: 24, top: 12),
+                  child: RichText(
+                    text: TextSpan(
+                      text: 'By proceeding, you agree to these ',
+                      style: Theme.of(context).textTheme.labelSmall!.copyWith(
+                            height: 1.5,
+                            color: const Color(0xFF8FA2B7),
+                          ),
+                      children: <TextSpan>[
+                        TextSpan(
+                          text: 'Terms and Conditions.',
+                          style: Theme.of(context)
+                              .textTheme
+                              .labelSmall!
+                              .copyWith(
+                                  height: 1.5,
+                                  color: const Color(0xFF5F97FF),
+                                  decoration: TextDecoration.underline,
+                                  decorationColor: const Color(0xFF5F97FF)),
+                        ),
+                      ],
+                    ),
+                  )),
+              const SizedBox(
+                height: 250,
+              ),
+            ],
+          ),
+        ),
+      ),
+      floatingActionButton: _disableButton
```

Comment:
Optimize this condition:- change to theme ElevatedButton and handle  the case disabled and active